lms/static/sass/course/_info.scss

```diff
@@ -56,6 +56,115 @@ div.info-wrapper {
       font-style: normal;
     }
 
+    div.upgrade-banner {
+      // This banner uses the Pattern Library's defined variables
+      @include border-left(0px);
+
+      border: 1px solid $uxpl-gray-background;
+      width: 100%;
+      display: table;
+
+      .notification-color-border {
+        width: 6px; //Value defined by UX team
+        min-height: 100%;
+        margin: 0;
+        display: table-cell;
+        background: $notification-highlight-border-color;
+      }
+
+      .notification-content {
+        display: inline-flex;
+        align-items: center;
+        align-content: flex-start;
+        flex-flow: row wrap;
+        background: $homepage-background;
+        width: 100%;
+        padding: $baseline/2 0;
+        margin-bottom: 0;
+        justify-content: space-between;
+
+        .upgrade-icon {
+          margin: 0;
+          padding: $baseline/2 $baseline;
+          flex-flow: row nowrap;
+          align-items: center;
+          flex: 7 1 50px;
+          // The following dimensions were added so that the
+          // icon will adjust as the notification is adjusted
+          // but will not be smaller or larger than UX requirements.
+          min-height: 50px;
+          min-width: 80px;
+          max-height: 90px;
+          max-width: 130px;
+
+          img{
+            min-height: 50px;
+            min-width: 80px;
+          }
+        }
+
+        .upgrade-msg {
+          flex: 5 1 60%; //This percentage was required to get the text
+                         // in the message to wrap when collapsed.
+          flex-direction: column;
+          margin: 0;
+          padding: 10px 0;
+          .msg-title {
+            font-weight: font-weight(semi-bold);
+            font-size: font-size(large);
+            line-height: $base-line-height;
+          }
+          .view-verified-info {
+            margin-top: $baseline / 4;
+            font-weight: font-weight(normal);
+            font-size: font-size(base);
+          }
+
+          a:link, a:hover, a:visited, a:active {
+            text-decoration: underline !important;
+          }
+        }
+
+        .upgrade-banner-button {
+          margin: 0;
+          padding: 10px 20px;
+          @include ltr {
```

Comment:
I'd remove these two sections, and change the original margin line to be:

    @include margin(0, $baseline/2, 0, 0)

In general, you should never need to use ``@include ltr`` because the helper functions provide all that you need. It is only necessary if there is a CSS rule that doesn't happen to have a helper.